tests/_benderjs/ckeditor/static/tools.js

```diff
@@ -1103,11 +1104,26 @@
 						throw new Error( 'Test named "' + specificTestName + '" already exists' );
 					}
 
-					outputTests[ specificTestName ] = ( function( testName, editorName ) {
-						return function() {
-							inputTests[ testName ]( bender.editors[ editorName ], bender.editorBots [ editorName ] );
-						};
-					} )( testName, editorName );
+					// We are creating new editor instance to isolate each test case (#1696).
+					if ( isolateTests && editorCounter++ ) {
```

Comment:
`editorCounter++` is called here and then the same `editorCounter++` is called while passing to iife function, which produces editors like `editor2`, `editor4`, `editor6`, etc instead of `editor2`, `editor3`, `editor4`, etc.